tests/unit/nginx-vhosts.bats

```diff
@@ -109,21 +109,11 @@ assert_error_log() {
   assert_output "0"
 }
 
-@test "(nginx-vhosts) nginx:build-config (with SSL CN mismatch)" {
-  setup_test_tls
-  deploy_app
-  assert_ssl_domain "node-js-app.dokku.me"
-}
-
-@test "(nginx-vhosts) nginx:build-config (with SSL CN mismatch & custom nginx template)" {
-  setup_test_tls
-  custom_ssl_nginx_template
-  deploy_app
-  assert_ssl_domain "node-js-app.dokku.me"
-}
-
```

Comment:
It's a BC that wouldn't affect existing apps though, right? IIRC the domains were added to the domains file so unless you create a new app and add SSL to it, you won't notice that this behaviour has changed.